### PR TITLE
Fixed some access for Plast Member at Blanks and removed interested from dropdown

### PR DESCRIPTION
**Pull request by nicoolle**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by christykobyletska — APPROVED** [review]

**Review by sharix4 — APPROVED** [review]

File: src/pages/userPage/Blanks/Blanks.tsx [hunk 1]
```diff
@@ -87,12 +87,10 @@ export const Blanks = () => {
         setExtractUPU(response.data);
         setExtractUPUFormat(getFormat(response.data.fileName));
     }
-
```
Comment (OleksandrBilan) [inline src/pages/userPage/Blanks/Blanks.tsx:90]: Why didn't you like this tabulation????

File: src/pages/userPage/Blanks/Blanks.tsx [hunk 2]
```diff
@@ -191,16 +196,16 @@ export const Blanks = () => {
...
-                                            {document.fileName}
-                                          </Paragraph>
+                                            : <Paragraph ellipsis={{ rows: 2, suffix: " " }}>
+                                                {document.fileName}
+                                            </Paragraph>
                                         }
 
                                     </div>
-                                    {(userToken.nameid === userId || IsUserHasAccessToManageBlanks(roles)) &&
+                                    {(userToken.nameid === userId || IsUserHasAccessToSeeAndDownloadBlanks(roles)) &&
```
Comment (OleksandrBilan) [inline src/pages/userPage/Blanks/Blanks.tsx:208]: 'IsUserHasAccess...'? Maybe it'd be better 'DoesUserHaveAccess...'

File: src/pages/userPage/Blanks/Blanks.tsx (hunk 1 above)
Comment (nicoolle) [reply src/pages/userPage/Blanks/Blanks.tsx:90]: It's not me, it's alt+shift+f ))

File: src/pages/userPage/Blanks/Blanks.tsx (hunk 2 above)
Comment (nicoolle) [reply src/pages/userPage/Blanks/Blanks.tsx]: Yes, much more. Done

**Review by andrii52 — APPROVED** [review]

**Pull request merged**